Django/auth_demo/users/views.py

```
@@ -0,0 +1,41 @@
+from django.shortcuts import render,redirect
+from django.contrib.auth import authenticate,login,logout
+from django.contrib import messages
+from django.contrib.auth.models import User
+
+# Create your views here.
+from django.shortcuts import render, redirect
```

Ruff (0.8.2)

Redefinition of unused `render` from line 1

Remove definition: `render`

(F811)